app/components/common/bubblePopover/index.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import Popover from "@material-ui/core/Popover";
+import { PopoverProps } from "@material-ui/core/Popover";
+import { withStyles } from "../../../helpers/withStylesHelper";
+const styles = require("./bubblePopover.scss");
+
+interface BubblePopoverProps extends PopoverProps {
+  menuItems?: React.ReactNode;
```

Comment:
You can omit this line.
At the below, BubblePopover class extends `React.Component<P>`. 
Then React Component interface inject React Component types.
so you can use `this.props.children` without any additional interface declaration.